cloudformation/apispec.yaml

```diff
@@ -0,0 +1,58 @@
+---
+openapi: 3.0.3
+info:
+  title: Xplorers API
+  version: 0.0.1
+  license:
+    name: MIT
+    url: https://opensource.org/licenses/MIT
+paths:
+  /:
+    post:
+      responses:
+          200:
+            description: 200 response
+            content:
+              application/json:
+                schema:
+                  $ref: "#/components/schemas/Empty"
+      x-amazon-apigateway-integration:
+        type: aws_proxy
+        httpMethod: POST
+        responses:
+          default:
+            description: XplorersBot Response
+            statusCode: 200
```

Comment:
Yes, API gateway invokes lambda asynchronously and immediately responds to slack with a 200 to avoid slack's 3 second timeout.

I understand that the correct response type in this scenario would be `202 Accepted` but slack expects a 200 response, more about this in slack docs [here](https://api.slack.com/interactivity/slash-commands#:~:text=Your%20app%20can%20do%20this%20simply%20by%20sending%20back%20an%20empty%20HTTP%20200%20response%20to%20the%20original%20request.).